content/en/guide/v10/web-components.md

```diff
@@ -50,6 +50,12 @@ function Foo() {
 }
 ```
 
+> [!Note]
+> Preact maps only camelCase JSX property names to the same camelCase JS properties of the custom elements,
+> and does not map dash-case JSX properties to camelCase JS properties like other JSX compilers or runtimes might.
+> If a custom element has a property named `someProperty`, using the syntax `someProperty={value}` will work while
+> `some-property={value}` will not (the latter will set the attribute instead).
```

Comment:
> [!Note]
> The `[!Note]` syntax results in a note like this one.

Reply:
I thought it did, but appears not to: https://deploy-preview-1156--preactjs.netlify.app/guide/v10/web-components#properties-and-attributes